go/storage/mkvs/db/badger/badger_test.go

```diff
@@ -0,0 +1,235 @@
+package badger
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/stretchr/testify/require"
+
+	"github.com/oasisprotocol/oasis-core/go/common"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/hash"
+	"github.com/oasisprotocol/oasis-core/go/storage/api"
```

Comment:
You should instead directly import the relevant package under `storage/mkvs`, the `api` package only re-exports.